code/__DEFINES/antagonists.dm

```diff
@@ -267,6 +267,10 @@ GLOBAL_LIST_INIT(human_invader_antagonists, list(
 /// Hijack is unavailable as a random objective below this player count.
 #define HIJACK_MIN_PLAYERS 30
 
+/// Hijack locations
+#define DEEP_SPACE 1
+#define SYNDICATE_BASE 2
```

Comment:
```suggestion
#define DEEP_SPACE "deep_space"
#define SYNDICATE_BASE "syndicate_base"
```

No reason to make these numbers

Reply:
There is a reason to make these numbers. get_desired_hijack_destination() uses greater than operations to determine which location a hijacker should send the shuttle in the case of having multiple antagonist datums.

Reply:
The alternative is to add extra logic to get_desired_hijack_destination() which would need to be updated every time a new location is added.